src/components/AccountContent/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import ProductAvailable from './ProductAvailable'
+import Review from './Review'
+
+const AccountContent = () => {
+  return (
+    <section className="account-content">
+      <ProductAvailable />
+      <Review />
```

Comment:
I think it shouldn't be only a standalone `Review` 🙋🏻‍♀️

Reply:
`AccountReview` or `ProductReview`, Which one is more comfortable for you?

Reply:
Do you mean `<ProductReview />` ? Plural is generally used for arrays or collections. Component name should not be pluralized.